bin/oref0-ns-loop.sh

```diff
@@ -169,7 +169,7 @@ function upload_recent_treatments {
 
 #nightscout cull-latest-openaps-treatments monitor/pumphistory-zoned.json settings/model.json $(openaps latest-ns-treatment-time) > upload/latest-treatments.json
 function format_latest_nightscout_treatments {
-    nightscout cull-latest-openaps-treatments monitor/pumphistory-zoned.json settings/model.json $(openaps latest-ns-treatment-time) > upload/latest-treatments.json
+    nightscout cull-latest-openaps-treatments settings/pumphistory-24h-zoned.json settings/model.json $(openaps latest-ns-treatment-time) > upload/latest-treatments.json
```

Comment:
This change is also needed in the openaps ns-loop, but i cannot test it myself as i don't use it.